apps/system/js/accessibility.js

```diff
@@ -284,11 +290,27 @@
     },
 
     /**
+     * Start a timeout that waits to display hints
+     * @memberof Accessibility.prototype
+     */
+    setHintsTimeout: function ar_setHintsTimeout(aHints) {
+      this.hintTimer = setTimeout(function onHintTimeout() {
```

Comment:
Also I just realized that it's probably best to use setInterval, clearInterval instead of setTimeout/clearTimeout. We want to keep suggesting an interaction in case the user did no hear it the first time.